tests/test_management.py

```diff
@@ -514,23 +653,42 @@ class TestLearnerDataTransmitIntegration(unittest.TestCase):
     Integration tests for learner data transmission.
     """
 
+    def setUp(self):
+        super(TestLearnerDataTransmitIntegration, self).setUp()
+
+        # pylint: disable=invalid-name
```

Comment:
@UmanShahzad Could you just disable `invalid-name` for the method as a whole? Then you wouldn't need to re-enable it in L681 below.

Reply:
:+1:

Reply:
Ok turns out I couldn't do it -- or at least I don't know how to scope it to just a method. If you've any idea I'll make the change -- google isn't helping here :p

Reply:
@UmanShahzad Hm, I guess I was assuming that putting the directive (is that what these lines are called?) right above the method definition would do it... If that's not the case then I'm not sure. Also tried googling this, to no avail. (Which makes me think we're missing something -- what we're trying to do doesn't seem super esoteric to me, so perhaps the reason why nobody else seems to be having problems with it is that it's actually super easy to accomplish? :))

Reply:
@UmanShahzad In any case, I'm fine with keeping this. No need to spend a lot of time trying to figure out how to change it.

Reply:
@itsjeyd There've been a lot of times I would like to control the scope of pylint more carefully (i.e., disable pylint for the next *5* lines only, etc.), but I think these features simply don't exist. It sounds like a good opportunity to contribute something ;)